src/main/java/ca/team2706/frc/robot/subsystems/DriveBase.java

```diff
@@ -454,6 +516,11 @@ public void setOpenLoopVoltageMode() {
      * Sets the talons to a disabled mode
      */
     public void setPositionNoGyroMode() {
+        if(!canRunAuto()) {
+            setDisabledMode();
+            return;
```

Comment:
I'm not sure I like the early return here. I would prefer an `if...else` structure.